src/string_bytes.cc

```diff
@@ -62,6 +62,10 @@ class ExternString: public ResourceType {
       return length_;
     }
 
+    // Maximum length in bytes that V8 can handle. 
+    // This equals to 268435440 bytes = (256 * 1024 * 1024) - 16
+    static const size_t kMaxLength = (1 << 28) - 16;
```

Comment:
Suggestion: mention in the comment that the constant mirrors `String::kMaxLength` from `deps/v8/src/objects.h`.